src/commons/types.ts

```diff
@@ -0,0 +1,25 @@
+import {ColorValue} from 'react-native';
+export interface Theme {
```

Comment:
Where did you get these keys from? I think there are tones of keys missing...

Reply:
I tried to look for usages in the code..
I've added more keys, let me know if something is still missing :)

Reply:
You can find most keys in `src/style.js`

Reply:
Tnx! added :)